test/e2e/e2e.test.ts

```diff
@@ -68,6 +68,10 @@ function getDeployCommand(): string {
     default:
       throw new Error(`Unknown platform: ${PLATFORM}`)
   }
+  if (process.env.RELEASE_BRANCH === 'on') {
```

Comment:
Do we really need to check this?
I think `return command + ` --version=${helper.getNewVersion()}`  is enough

Reply:
Not sure but with --version=next throw:
```log
    Command failed with exit code 2: /home/runner/work/chectl/chectl/bin/run server:deploy --batch --platform=minikube --installer=olm --telemetry=off --chenamespace=eclipse-che --che-operator-cr-patch-yaml=test/e2e/resources/cr-patch.yaml --multiuser --skip-cluster-availability-check --version=next
    (node:9312) [ENOENT] Error: spawn chectl ENOENT
    (Use `node --trace-warnings ...` to show where the warning was created)
        Error: Command server:deploy failed. Error log: 
        /home/runner/.cache/chectl/error.log. Eclipse Che logs: 
        /tmp/chectl-logs/1629896067163.
     ❕olm-suggested-namespace flag is turned on. Eclipse Che will be deployed in namespace: eclipse-che.
    [12:54:27] Verify Kubernetes API [started]
    › Current Kubernetes context: 'minikube'
    [12:54:27] Verify Kubernetes API...OK [title changed]
    [12:54:27] Verify Kubernetes API...OK [completed]
    [12:54:27] 👀  Looking for an already existing Eclipse Che instance [started]
    [12:54:27] Verify if Eclipse Che is deployed into namespace "eclipse-che" [started]
    [12:54:27] Verify if Eclipse Che is deployed into namespace "eclipse-che"...it is not [title changed]
    [12:54:27] Verify if Eclipse Che is deployed into namespace "eclipse-che"...it is not [completed]
    [12:54:27] 👀  Looking for an already existing Eclipse Che instance [completed]

      at makeError (node_modules/execa/lib/error.js:60:11)
      at handlePromise (node_modules/execa/index.js:118:26)
```

Reply:
ok, @mmorhun after conversations with Mykola seems we cannot deploy next via --version flag

Reply:
```javascript
helper.getNewVersion()
```
return `next` if doesn't exist VERSION file